src/components/Form.tsx

```diff
@@ -128,7 +138,7 @@ function FormInput(props: IFormInputProps) {
       <input
         type={props.type}
         disabled={props.disabled ?? false}
-        className="block w-full mt-2 px-2 py-1 bg-gray-800 text-white border-slate-700 rounded-md shadow-sm focus:border-indigo-700 focus:ring focus:ring-indigo-700 focus:ring-opacity-5 disabled:text-gray-600 disabled:placeholder:text-gray-600"
+        className="block w-full px-2 py-1 mt-2 text-white bg-gray-800 rounded-md shadow-sm border-slate-700 focus:border-indigo-700 focus:ring focus:ring-indigo-700 focus:ring-opacity-5 disabled:text-gray-600 disabled:placeholder:text-gray-600"
```

Comment:
Can you please remove these extra changes? It will help in reviewing. Thanks!

Reply:
You have to revert all of these changes. I am not sure how they appeared.

Reply:
Nope :)

Reply:
Ok.. So, let me understand. There are two same lines of code showing, one marked with red that means they have been removed and one marked with green that means they have been addded. Right?

So I need to just delete those green marked lines that were added?

I am a new to the language as well as open source.  So, please make me understand

Reply:
Yes, red shows removal, and green shows addition. In this case, it seems like the order of the classes has been changed, perhaps by a formatter. Usually, in a PR, extra unnecessary changes are bad because they hinder the review process. Here, since only one file has these changes, you can simply copy the current state of the file from our repository and paste it there.

Reply:
Oh.. I got it. I had an extension installed which changed the order. I am correcting it now. thank you so much.